docs/content/en/install/_index.md

```diff
@@ -1,7 +1,21 @@
 ---
-title: "Install K8ssandra"
-linkTitle: "Install"
+title: "Installs"
+linkTitle: "Installs"
 weight: 3
+no_list: true
 description: >
-  Topics for installing K8ssandra on local (dev) machines or production-level K8ssandra clusters on a variety of cloud platforms.
+  Orientation of install topics for v1 K8ssandra and v2 K8ssandra Operator.
 ---
+
+Welcome! Starting with the introduction of the unified K8ssansdra Operator in March 2022, the install topics are organized as follows:
+
+* For details about the new K8ssandra Operator, see this [local install](https://docs-staging-v2.k8ssandra.io/install/local/) topic. It showcases the K8ssandra Operator support for single- and multi-cluster environments. Examples use local **kind** Kubernetes along with helper scripts invoked by `make` commands, as well as `helm` and `kustomize` options. Topics covering installs that use K8ssandra Operator for Google GKE, Amazon EKS, and Azure AKS may be added at a later time. 
+
+* For details about using the original K8ssandra (1.4.x) project, start on this [introductory page](https://docs-staging-v1.k8ssandra.io/install/). It provides links to the following topics for single-cluster installs:
+  * [Local K8s](https://docs-staging-v1.k8ssandra.io/install/local/)
```

Comment:
You could probably go ahead and start pointing things to docs-v1 or docs-v2 and we can publish those prod sites whenever.  That will make your life easier going forward here.